modules/navier_stokes/src/userobjects/PINSFVRhieChowInterpolator.C

```diff
@@ -100,16 +100,6 @@ PINSFVRhieChowInterpolator::pinsfvSetup()
   Moose::FV::interpolateReconstruct(
       _smoothed_eps, _eps, _smoothing_layers, false, _geometric_fi, *this);
   ADReal::do_derivatives = saved_do_derivatives;
-
-  _eps.assign(_smoothed_eps);
-
-  const auto porosity_name = deduceFunctorName(NS::porosity);
-  for (const auto tid : make_range((unsigned int)(1), libMesh::n_threads()))
-  {
-    auto & other_epss = const_cast<Moose::Functor<ADReal> &>(
-        UserObject::_subproblem.getFunctor<ADReal>(porosity_name, tid, name()));
-    other_epss.assign(_smoothed_eps);
-  }
```

Comment:
If you want to be able to retrieve the smoothed porosity in consumer objects, you will need to add the smoothed porosity to the subproblem's collection of functors.

Reply:
yeah and we need to keep track of them in a _epss array too because that s where some constructs retrieve it

should be good now